config/settings/local.py

```diff
@@ -10,14 +10,24 @@
 import environ
 BASE_DIR = environ.Path(__file__) - 3  # (/a/myfile.py - 2 = /)
 CONFIG_DIR = BASE_DIR.path('config')
+env = environ.Env()
 environ.Env.read_env(str(CONFIG_DIR.path('env/.local')))
 
 from .common import *  # noqa
-
+"""
 EMAIL_HOST = 'localhost'
 EMAIL_PORT = 1025
+"""
+# EMAIL
+EMAIL_HOST = env.str('EMAIL_HOST', 'smtp.sendgrid.com')
+EMAIL_HOST_PASSWORD = '4hz7wwr09978' #env('SENDGRID_PASSWORD')
```

Comment:
I think you committed sendgrid secrets here. We should delete the resource on heroku and provision a new one to get a new password. Also remove these.